docs/telemetry.md

```diff
@@ -77,6 +77,9 @@ class OpenTelemetryProvider {
   }
 }
 
+// Before DLO loads and initializes
+window['_dlo_telemetryProvider'] = new OpenTelemetryProvider();
```

Comment:
I missed this assignment to DLO configuration in the previous PR to add telemetry documentation.